src/frontend/magnify/src/services/http/HttpService.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { rest } from 'msw';
+import { server } from '../../mocks/server';
+import { UsersRoutes } from '../../utils/routes/api/users.routes';
+import { buildApiUrl, HttpService, MagnifyApi } from './http.service';
+
```

Comment:
the test filename doesn't match the service file name, `http.service.ts` / `HttpService.test.tsx`, conventions tell to name the test file like `<originalName>.spec(|test).tsx`

Reply:
The same goes for other files